layouts/auto_generated_product.tpl

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+{% include "template-variables" %}
+<html class="content-page {% if editmode %}editmode{% else %}public{% endif %}" lang="{{ page.language_code }}">
+<head prefix="og: http://ogp.me/ns#">
+  {% include "html-head" %}
+  {% include "edicy-tools-variables" %}
+</head>
+
+{%- capture bottom_content_html -%}
+  {%- unless editmode -%}
+    {%- content bind=product name="content" -%}
+  {%- endunless -%}
+{%- endcapture -%}
+
+{%- assign bottom_content_size = bottom_content_html | strip | size -%}
+
+{%- capture gallery_content_html -%}
+  {%- unless editmode -%}
+    {%- content bind=product name="gallery" -%}
+  {%- endunless -%}
+{%- endcapture -%}
+
+{%- assign gallery_content_size = gallery_content_html | strip | size -%}
+
+<body class="item-page main-menu-not-fitting{% if site.search.enabled %} search-enabled{% endif %}">
+  {% include "template-svg-spritesheet" %}
+
+  <div class="holder">
+    {% include "header" %}
+
+    <main class="main" data-search-indexing-allowed="true">
+      <div class="main-inner">
+        <div class="content-wrap">
+          {% if page.level != 0 %}
+            {% include "submenu-for-current" exclude_products: true %}
+          {% endif %}
```

Comment:
`submenu-for-current` is not needed as subpages cannot be added to auto-rendered product pages